### PR TITLE
Jakob/update schedule

### LINKED ISSUE #388: Add new workshops
Update workshops page to have this year's workshops

### PR DESCRIPTION
**Pull request by jakobreinwald**
Fixes #388

### CODE REVIEW

File: src/components/WorkshopPage/WorkshopPage.js [hunk 1]
```diff
@@ -71,57 +71,48 @@ const workshops = [
...
+				to the foundations of Web APIs that will allow you to integrate them into any fullstack project. We 
+				will be covering the client-server model, HTTP requests & responses, and how we can leverage external 
+				servers to retrieve data and services for an app. Keep reading to learn more!`,
+				youtube: 'https://youtu.be/ZIamyNtujBY?si=r0v5ojIThiKzzcsQ',
+				readme: 'https://github.com/uclaacm/hothXI-workshops/tree/main/intro-to-webapis',
+				slides: 'https://docs.google.com/presentation/d/1CpiERteBZsqKqRjnYgGruBO2aZnonjRWibSqnthcRws/edit?usp=drive_link'
 			},
 			{
 				title: 'Intro to Servers',
 				author: 'Andy Lewis',
```
Comment (katelynsyu) [inline src/components/WorkshopPage/WorkshopPage.js:96]: I think `Intro to Servers` is taught by Sam.

File: src/components/WorkshopPage/WorkshopPage.js [hunk 2]
```diff
@@ -148,76 +138,67 @@ const workshops = [
...
-				websites hella based) :)`,
-				youtube: 'https://youtu.be/l6D8X0L_L_g',
-				readme: 'https://github.com/uclaacm/hothX-workshops/tree/main/intro-to-databases',
-				slides: 'https://docs.google.com/presentation/d/1LE2CYSIvGKSbGdGuZuMC1hOnxjmKgP3uwH0-LPEWSFw/edit?usp=sharing'
+				author: 'Sneha Agarwal',
+				description: `Hi! Welcome to HOTHXI's Database workshop, in this workshop we'll be covering
+				the basics of what a database is and walkthrough how to set up, retrieve information, and
+				add information to our database! Before starting this workshop I recommend checking out 
+				the React JS or React Native workshops if you're not familar with them.`,
+				youtube: 'https://youtu.be/JZrRLf5Gmqg?si=BVWuiqDqKvcMYTxH',
```
Comment (katelynsyu) [inline src/components/WorkshopPage/WorkshopPage.js:168]: I think this links to Kayla's workshop

File: src/components/WorkshopPage/WorkshopPage.js [hunk 3]
```diff
@@ -148,76 +138,67 @@ const workshops = [
...
+				title: 'Intro to Deep Learning',
+				author: 'Leon Lenk, Maxine Wu, & Jordan Lin (ACM AI)',
+				description: `Shoutout to the folks over at ACM AI for putting this together! They created this workshop
+				so serve as an introduction to deep learning, including a notebook linked on the readme to show you 
+				actual deep learning code and take you through a hands-on of how you can build this code yourself! 
+				In the process of building this model, they go over Big Data, avoiding bias and unintended patterns, 
+				understanding our data, making the network, and training`,
+				youtube: 'https://youtu.be/RggC7sb1_cE?si=4FLeRtfCLOgZezhU',
+				readme: 'https://bit.ly/acm_ai_24w_intro_dl',
+				slides: null
```
Comment (katelynsyu) [inline src/components/WorkshopPage/WorkshopPage.js:182]: In the deploy preview, there's still a button for slides, even though it's value is `null`.

**Review by katelynsyu — CHANGES_REQUESTED** [review]
a few things with the workshop page are off, but they should be a quick fix.

File: src/components/WorkshopPage/WorkshopPage.js (hunk 3 above)
Comment (katelynsyu) [reply src/components/WorkshopPage/WorkshopPage.js:182]: to fix this, i think we have to go into `Workshop.js` line [73](https://github.com/uclaacm/hoth.uclaacm.com/blob/9a9b64af26ad9c73fcd1d13caba1e2be18b2c8ed/src/components/WorkshopPage/Workshop.js#L73) and add a check if there's a null passed in like there is for the README value in line [65](https://github.com/uclaacm/hoth.uclaacm.com/blob/9a9b64af26ad9c73fcd1d13caba1e2be18b2c8ed/src/components/WorkshopPage/Workshop.js#L65)

**Review by nathanzzhang — APPROVED** [review]
yeehaw LGTM!!!

**Review by katelynsyu — APPROVED** [review]
lgtm!

**Pull request merged**